utils/fixed_decimal/src/decimal.rs

```diff
@@ -422,7 +436,11 @@ impl FixedDecimal {
     /// assert_ne!(zero, negative_zero);
     /// ```
     pub fn negate(&mut self) {
```

Comment:
Suggestion: Keep this method, but add another method below it called `set_sign(&mut self, sign: Sign)`, and the chainable version `with_sign(mut self, sign: Sign) -> Self`. Otherwise there is no programmatic API for the user to opt in to a particular sign other than using the string parsing function.

Reply:
I like `set_sign` and `with_sign`.
I’m actually not sure we should keep `negate`; it feels too much like a numeric operation for something which isn’t really a numeric type (in particular the choice of `None` vs. `Positive` on a negated negative is arbitrary).
I don’t know what our policy is when it comes to deprecating bits of the API though.

Reply:
It's still pre-1.0 so we can change things to our heart's content. I'm okay if you remove the `negate` function.